clientv3/watch.go

```diff
@@ -194,6 +199,15 @@ func (w *watcher) Watch(ctx context.Context, key string, opts ...OpOption) Watch
 		}
 	}
 
+	select {
```

Comment:
`errc` was only meant to be consumed by `Close` and `run`; it's unreliable to use it like this